docs/reference/parts.rst

```diff
@@ -0,0 +1,14 @@
+.. _parts:
+
+Parts
+*****
+
+Parts, powered by :external+craft-parts:ref:`craft-parts <reference>`, power the build
```

Comment:
I'm not sure whether this is new behaviour, but some of the `:ref:` and `:external+` directives are appending a fragment to the URL when none was specified. In this case the link is:

```
https://canonical-craft-parts.readthedocs-hosted.com/en/latest/reference/index.html#reference
```